src/OpenSage.Game/Gui/Apt/ActionScript/Opcodes/Arithmetic.cs

```diff
@@ -24,7 +24,10 @@ public sealed class Subtract : InstructionBase
 
         public override void Execute(ActionContext context)
         {
-            throw new NotImplementedException();
+            var a = context.Stack.Pop();
+            var b = context.Stack.Pop();
+
+            context.Stack.Push(Value.FromFloat(b.ToFloat() - a.ToFloat()));
```

Comment:
Is all apt arithmetic in floating point?

Reply:
Yes. NonNumerical values default to 0 (which ToFloat *should* do, not sure here tbh). See https://www.adobe.com/content/dam/acom/en/devnet/pdf/swf-file-format-spec.pdf **ActionSubtract**